db/migrations/0003_alter_order_created_at.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.2 on 2024-11-21 18:18
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_user_order_alter_movie_actors_alter_movie_genres_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='order',
+            name='created_at',
+            field=models.DateTimeField(default=django.utils.timezone.now),

```

Comment:
The `created_at` field is now set to default to the current time using `django.utils.timezone.now`, which is a common practice for timestamp fields to automatically record creation time.